doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -130,7 +130,11 @@ Bluetooth mesh samples
 nRF9160 samples
 ---------------
 
-|no_changes_yet_note|
+* :ref:`modem_shell_application` sample:
+
+  * Added:
+
+    * nRF 9160 DK overlays for enabling BT support. When running this configuration, BT scanning and advertising can be mabe with ``bt`` command.
```

Comment:
As a general comment, `nRF9160` should be written without a space. The same change should be done for the commit message.

Typo: "mabe"

Reply:
Fixed.